src/compas_rhino/scene/meshobject.py

```diff
@@ -57,6 +53,9 @@ def __init__(self, mesh, disjoint=False, **kwargs):
         self._guids_facelabels = None
         self._guids_spheres = None
         self._guids_pipes = None
+        self.vertexgroup = vertexgroup
+        self.edgegroup = edgegroup
+        self.facegroup = facegroup
```

Comment:
Don't have to be this PR, but one thing to consider would be also allow to serialise the context specific parameters.